pkg/task/backup.go

```diff
@@ -213,6 +213,7 @@ func RunBackup(c context.Context, g glue.Glue, cmdName string, cfg *BackupConfig
 
 	err = client.SaveBackupMeta(ctx, ddlJobs)
 	if err != nil {
+		log.Error("save backup meta failed", zap.Reflect("meta", ddlJobs))
```

Comment:
The number of ddl jobs may be particularly large, especially as the system runs longer and longer, it is recommended to print only version information